src/__tests__/transaction.test.ts

```diff
@@ -10,3 +11,15 @@ describe("Transaction object", () => {
         expect(transaction).toBeInstanceOf(Transaction);
     });
 });
+
+describe("Transaction wrapper object", () => {
+    it("Should instantiate with valid parameters", () => {
+        const transactionWrapper = new TransactionWrapper(
+            new Transaction(TransactionType.COIN, "RECIPIENT_KEY", "SENDER_KEY", 1, new Date()),
```

Comment:
What does this 1 signify here? Might be better to create a constant for this number.